src/utils/index.ts

```diff
@@ -35,157 +34,70 @@ export const setTimeOut = async (): Promise<void> => {
 
 // ************************************************************ //
 
-export const mapData = (data: any[] = [], keys: string[]) => data.map( (d: any) =>
-    keys.reduce((o, key) => ({ ...o, [key]: d[key] }), {})
-);
-
-// ************************************************************ //
-
-export const getInfo = async (what: string, dataKey: string, dataUrl: string, page = 1): Promise<void> => {
-    const url = `${dataUrl}?page=${page}&per_page=${PER_PAGE}`;
-    try {
-        const jsonData = await fetch(url, HEADERS);
-        const data = await jsonData.json();
-        const dataLength = data?.length || 0;
+export const setTeamsData = async (teams: GitHubTeams[]): Promise<GitOrgData> => {
+    const obj = { "list": [], "details": {} } as GitOrgData;
```

Comment:
`const obj` is a local variable name repeated in each of teams, members and repos setter functions. I would give it a more semantic name in each function, eg. `const reposObj` or `const reposFields` or anything better